pkgs/applications/misc/zathura/core/default.nix

```diff
@@ -25,11 +31,12 @@ stdenv.mkDerivation rec {
     "RSTTOMAN=${docutils}/bin/rst2man.py"
     "VERBOSE=1"
     "TPUT=${ncurses.out}/bin/tput"
-  ] ++ lib.optional synctexSupport "WITH_SYNCTEX=1";
+    (optionalString synctexSupport "WITH_SYNCTEX=1")
+  ];
 
   postInstall = ''
     wrapProgram "$out/bin/zathura" \
-      --prefix PATH ":" "${lib.makeBinPath [ file ]}" \
+      --prefix PATH ":" "${file}/bin" \
```

Comment:
Well, a slow migration to `makeBinPath` will reduce pain even in the single-package case when for whatever reason `file` gets multiple outputs (which makes some sense).

So yes, I would be OK with not adding `makeBinPath` initially, but if it is there, I want it to stay.